### PR TITLE
Feat/popup

### PR DESCRIPTION
**Pull request by vozbrann**

### CODE REVIEW

**Comment by vlacher12** [comment]
image

File: src/components/popup/style.ts [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+export const StyledPopupBackdrop = styled.div`
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: 1040;
+  width: 100vw;
+  height: 100vh;
+  background-color: #000;
+  opacity: 0.5;
+`;
```
Comment (vlacher12) [inline src/components/popup/style.ts:78]: Use `$secondary` color from schema with 0.15 opacity

Comment (vlacher12) [inline src/components/popup/style.ts:78]: Pls order properties in alphabetical order

File: src/components/popup/README.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+| propName     | propType        | defaultValue | isRequired |
+| ------------ | --------------- | ------------ | ---------- |
+| children     | React.ReactNode | -            | -          |
+| ref          | React.RefObject | -            | -          |
+| title        | string          | -            | -          |
+| description  | string          | -            | -          |
+| onHide       | (e) => void;    | -            | +          |
+| show         | boolean         | -            | +          |
+| clickOutHide | boolean         | true         | -          |
```
Comment (vlacher12) [inline src/components/popup/README.md:56]: Could you pls rename this prop?

**Comment by vlacher12** [comment]
@vozbrann could you pls add `exit on ESC button` functionality?

**Comment by vlacher12** [comment]
@vozbrann could you pls pull `master` branch and upgrade package version?

**Review by vlacher12 — APPROVED** [review]

**Review by KotAleksei — APPROVED** [review]

**Review by lazzystep — APPROVED** [review]

**Pull request merged**